sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -150,12 +179,142 @@ def test_pin_joint_chaos_pendulum():
                                        (h/4 + lB/2)*omega)*A.x
 
 
+@XFAIL
```

Comment:
@sidhu1012 I thought based on #22304 that this issue would be fixed, but it still fails when using the symengine. Could you give some clarification?

Reply:
This https://github.com/sympy/sympy/issues/22304#issuecomment-1085461259 says that the issue is fixed, earlier it wasn't sympy's error, I think using latest symengine version should fix this.

Reply:
After [this commit](https://github.com/sympy/sympy/pull/23920/commits/30d833b1d3e5a0703ef2d7da6681d23f1fbd75f5) I've introduced Xfail, but before that it was still failing on the latest symengine version (I tested 0.9.2 local).

Reply:
@sidhu1012 It seems it is still not fixed see [#22304 comment](https://github.com/sympy/sympy/issues/22304#issuecomment-1239321187)